packaging/redhat/fty-alert-list.spec

```diff
@@ -1,6 +1,5 @@
 #
 #    fty-alert-list - Provides information about active alerts
-# Note: this file was amended after generation, take care to keep that
```

Comment:
Do we still have some customization preserved? Then better not remove the notice :)